posthog/tasks/scheduled.py

```diff
@@ -47,6 +47,7 @@
     update_quota_limiting,
     verify_persons_data_in_sync,
 )
+from posthog.tasks.stop_surveys_reached_target import stop_surveys_reached_target
```

Comment:
I wonder why all other tasks are imported from tasks.py. Maybe I need to do so too?

Reply:
Yeah please do! Like this one -> https://github.com/PostHog/posthog/blob/master/posthog/tasks/tasks.py#L533 (keep the `stop_surveys_reached_target` file)